content/test/integration/controller/active-pointers.spec.ts

```diff
@@ -178,6 +178,65 @@ loadStandaloneTestEnvironment()('Integration - Create entities', (testEnv) => {
     expect(queryResult.rows[1].pointer).toBe('1,0')
   })
 
+
+  it('when overwriting multiple scenes, unused pointers should be deleted from active-pointers table', async () => {
+    // Create scene
+    let form = createForm('bafkreigaea5hghqlq2462z5ltdaeualenzjtm44xl3hhog4lxzoh7ooliy', 'scene_original.json');
```

Comment:
Do you mind adding a comment explaining the pointers deployed on each line? So it's easier to understand the test